pkg/helper/autonode/helpers.go

```diff
@@ -129,11 +153,11 @@ func InteractivePrompt(r *rosa.Runtime, cmd *cobra.Command, cluster *cmv1.Cluste
 	}
 
 	// Set mode to enabled
-	autonodeMode = "enabled"
+	mode = "enabled"
```

Comment:
I would prefer this to be `autoNodeMode` like it was; purely because there are CLI operating modes we call `mode` (auto mode + interactive mode`

I think I brought that up in a past comment, but we avoid calling it something as generic as `mode` since that var name is used in every command to signify CLI operating mode

Reply:
For example, in `create/cluster`:

```go
if isSTS && mode != "" {
		isValidMode := arguments.IsValidMode(interactive.Modes, mode)
		if !isValidMode {
			r.Reporter.Errorf("Invalid --mode '%s'. Allowed values are %s", mode, interactive.Modes)
			os.Exit(1)
		}
	}
```

Reply:
Updated on this commit [9f4c0b0](https://github.com/openshift/rosa/pull/3020/commits/9f4c0b096510fc069cf2531425b3e395b5c4d507)